src/gluttony/record/consumer.clj

```diff
@@ -9,7 +9,7 @@
    [gluttony.record.message :as r.msg]
    [gluttony.sqs :as sqs])
   (:import
-   (cognitect.aws.client
```

Comment:
This ns is moved at https://github.com/cognitect-labs/aws-api/commit/e262aac7a5cd2c1214a632ba851f95b6686a9b0f#diff-be06d82173b4b88bcf4f60c30239fb9ede8dd25bf113da07259435b019069652R4